src/pages/christmas-2021.tsx

```diff
@@ -0,0 +1,111 @@
+import MainLayout from "@components/pages/MainLayout";
```

Comment:
Can you make this a component and temporarily replace the current hero on homepage with this?

Reply:
Will do!